boards/arm/stm32_f4xx/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# STM32_F4XX board configuration
+
+# Copyright (c) 2016 Linaro Limited.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Add your © as well

Reply:
Just existing files copied and only minor tweaking done here, I was thinking leaving the copyrights as is.

Reply:
On my side I'm pushing people that push boards to only put their own copyright on all files.
For sure this is always copy/paste of a previous board, but if we apply the same logic throughout the project life:
- Keeping the initial copyright, we end up with the same copyright for all boards files. But counting on the endless flow of small changes on this generic code the current files have nothing to do with the initial ones on which the copyright initially applied
- Keeping the initial copyright and adding your own, we end up with a list of copyright that reflect the copy/paste history but has no specific interest.

Using your copyright, we recognize the ownership of the one that did the effort of the porting (which include copy/paste but also validation and upstream effort).